.rubocop.yml

```diff
@@ -0,0 +1,17 @@
+inherit_from: .rubocop_todo.yml
```

Comment:
Yeah, I'm having some trouble getting this into a usable state, integrating with the latest commits from the foundation-6 branch. I may just scrap this fork and start over, and given @potomak's comments from today, likely refactor this to be a breaking change to display_flash_messages.

Probably have to wait until tomorrow though. My kids are home from school early today due to a broken water main at the school. No toilets, no school!